lib/Dialect/Torch/Utils/Utils.cpp

```diff
@@ -78,6 +78,18 @@ Type Torch::getTypeForScalarType(
   }
 }
 
+Type Torch::getTorchTypeForScalarType(MLIRContext *context,
+                                      torch_upstream::ScalarType dtypeInt) {
+  switch (dtypeInt) {
+  case torch_upstream::ScalarType::Double:
+    return Torch::FloatType::get(context);
+  case torch_upstream::ScalarType::Long:
+    return Torch::IntType::get(context);
+  default:
+    assert(false && "Unsupported scalar type to Torch type conversion");
```

Comment:
nit: I think it might be more appropriate to use `llvm::report_fatal_error`, since there are still missing `case`s. The way I understand it, `assert`s are for testing invariants of the function, but in this case the `default` branch can be reached without breaking any invariants, such as by passing `ScalarType::Bool` as the `dtypeInt` argument

Reply:
Good point. Changed as suggested.